examples/coordinates/plot_mars-coordinate-frame.py

```diff
@@ -0,0 +1,143 @@
+r"""
+============================================
+Create a new coordinate frame class for Mars
+============================================
+
+This example describes how to subclass and define a custom coordinate frame for a
+planetary body which can be described by a geodetic or bodycentric representation,
+as discussed in :ref:`astropy:astropy-coordinates-design` and
+:ref:`astropy-coordinates-create-geodetic`.
```

Comment:
I think we should add here something like: "Note that we use the frame here only to store coordinates. To use it to determine where, e.g., to point a telescope on Earth to observe Olympus Mons, one would need to add the frame to the transfer graph, which is beyond the scope of this example."